magnetapiclient.test.js

```diff
@@ -0,0 +1,176 @@
+const expect = require('expect');
+const nock = require('nock'); 
+
+var MagnetAPIClient = require('./magnetapiclient');
+
+
+
+describe('MagnetAPIClient', function () {
+
+    const ENDPOINT_URI = "http://magnetapi.klangoo.com/NewsAgencyService.svc";
+    const CALK = "ENTER_YOUR_CALK";
+    const SECRET_KEY = "ENTER_YOUR_SECRET_KEY";
+    const articleUID = "2017/03/28/example-api-usage";
+
+    let client;
+    let api;
+
+    beforeEach(function () {
+
+        client = new MagnetAPIClient(ENDPOINT_URI, CALK, SECRET_KEY);
+        api = nock(ENDPOINT_URI, {
+            reqheaders: {
+                'content-type': 'application/x-www-form-urlencoded'
+            }
+        });
+    });
+
+    it('should be able to add an article', function (done) {
+
+        const request = {
+            text: 'SAMPLE ARTICLE TEXT',
+            title: 'SAMPLE ARTICLE TITLE',
+            insertDate: '23 JAN 2017 10:12:00 +01:00',
+            url: 'http://demo.klangoo.com/article-demo/api-example',
```

Comment:
Only "text" or "url" is accepted in request by Magnet. Using both will cause Magnet API throw error.

Reply:
Hmm that is good to know though their sample code that this test is based off of did the same thing: https://github.com/cnnlabs/MagnetApiClient.Node/blob/master/newsagencysample.js#L16